analyzers/tests/SonarAnalyzer.UnitTest/CFG/Roslyn/RoslynControlFlowGraphTest.cs

```diff
@@ -85,5 +85,27 @@ public void Method()
             var anonymousFunction = cfg.Blocks.SelectMany(x => x.Operations).SelectMany(x => x.DescendantsAndSelf()).OfType<FlowAnalysis.IFlowAnonymousFunctionOperation>().Single();
             cfg.GetAnonymousFunctionControlFlowGraph(IFlowAnonymousFunctionOperationWrapper.FromOperation(anonymousFunction)).Should().NotBeNull();
         }
+
+        [TestMethod]
+        public void FlowAnonymousFunctionOperations_FindsAll()
+        {
+            const string code = @"
+public class Sample {
+    private System.Action<int> Simple(int a)
+    {
+        var x = 42;
+        if (a == 42)
+        {
+            return (x) => {  };
+        }
+        return (x) => {  };
```

Comment:
Let's have one without parenthesized. Just in case.
```suggestion
        return x => {  };
```